### PR TITLE
BUGFIX: SD-3022 fixing typos dataset

### PR DESCRIPTION
**Pull request by dlcd2203**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/199/ : SUCCESS in 14s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-oss-check/206/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/200/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-oss-check/207/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/202/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-oss-check/208/ : SUCCESS in 3s

File: ecommerce-demo/workspaces/demo/ldm.json [hunk 1]
```diff
@@ -147,7 +147,7 @@
       },
       {
         "id": "monthlyinventory",
-        "title": "Monthlyinventory",
+        "title": "Monthly inventory",
         "description": "Monthlyinventory",
```
Comment (anvuong119) [inline ecommerce-demo/workspaces/demo/ldm.json:151]: Should we change `"Monthlyinventory"` → `"Monthly inventory"` as well?

Also, should we change `"Monthlyinventory"` → `"Monthly inventory"` at the following places as well, for the sake of consistency?

- https://github.com/gooddata/gooddata-public-demos/blob/5554bda5aecb596bb9685cb421b2a733358cf1ff/ecommerce-demo/workspaces/demo/ldm_v2.5.0.json#L143

- https://github.com/gooddata/gooddata-public-demos/blob/5554bda5aecb596bb9685cb421b2a733358cf1ff/ecommerce-demo/workspaces/demo/ldm_v2.5.0.json#L206

File: ecommerce-demo/workspaces/demo/workspaceAnalytics.json [hunk 2]
```diff
@@ -8634,11 +8634,11 @@
...
           "titlePattern": "%titleBase - %granularityTitle"
         },
-        "id": "imventory_month",
+        "id": "inventory_month",
         "tags": [
-          "Imventory month"
+          "Inventory month"
         ],
-        "title": "Imventory month"
+        "title": "Inventory month"
```
Comment (anvuong119) [inline ecommerce-demo/workspaces/demo/workspaceAnalytics.json:8641]: Should we change `"Monthlyinventory"` → `"Monthly inventory"` at the following places as well, for the sake of consistency?

- https://github.com/gooddata/gooddata-public-demos/blob/5554bda5aecb596bb9685cb421b2a733358cf1ff/ecommerce-demo/workspaces/demo/workspaceAnalytics.json#L7983

- https://github.com/gooddata/gooddata-public-demos/blob/5554bda5aecb596bb9685cb421b2a733358cf1ff/ecommerce-demo/workspaces/demo/workspaceAnalytics.json#L8057

- https://github.com/gooddata/gooddata-public-demos/blob/5554bda5aecb596bb9685cb421b2a733358cf1ff/ecommerce-demo/workspaces/demo/workspaceAnalytics_aio.json#L7498

- https://github.com/gooddata/gooddata-public-demos/blob/5554bda5aecb596bb9685cb421b2a733358cf1ff/ecommerce-demo/workspaces/demo/workspaceAnalytics_aio.json#L7561

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/203/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-oss-check/210/ : SUCCESS in 3s

**Comment by anvuong119** [comment]
LGTM

**Review by huavannis — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result
 - No issue found. Congratulations!

More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-public-demos-gate-PR87

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/gooddata-public-demos/job/gooddata-public-demos-validation/205/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-merge-compliance-check/77/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-sonarqube-scan-zuul-gate/75/ : SUCCESS in 36s
- https://checklist.intgdc.com/job/compliance/job/gooddata-public-demos-fossa-scanning-zuul/38/ : FAILURE in 5s (non-voting)

**Pull request merged**